src/components/MVP/AnnouncementCard/AnnouncementCard.module.scss

```diff
@@ -25,12 +27,20 @@
         url(/img/bday.webp);
       background-size: cover;
     }
+    .usa-card-group + h2 {
+      margin-top: units(6) !important;
+    }
     .usa-card__container {
       &:hover {
         background-blend-mode: soft-light;
         border-color: color('primary-light');
       }
     }
+    .usa-tag {
```

Comment:
Not sure why this is reverting back, I must have some funky commits happening. Will get it fixed up.